angular-material/angular-material-tests.ts

```diff
@@ -4,7 +4,8 @@ var myApp = angular.module('testModule', ['ngMaterial']);
 
 myApp.config((
     $mdThemingProvider: ng.material.IThemingProvider,
-    $mdIconProvider: ng.material.IIconProvider) => {
+    $mdIconProvider: ng.material.IIconProvider,
+	$mdProgressCircularProvider: ng.material.IProgressCircularProvider) => {
```

Comment:
I think what you want is `git rebase --onto master types-2.0`. See [this question](http://stackoverflow.com/questions/40524681/why-git-rebase-shows-conflicts-in-the-files-i-did-not-modify).